packages/sync/src/Functions.php

```diff
@@ -368,6 +373,10 @@ public static function roles() {
 	 * @return string
 	 */
 	public static function get_timezone() {
+		// @todo: Only return wp_timezone_string once WP 5.3 is the minimum version.
+		if ( function_exists( 'wp_timezone_string' ) ) {
+			return wp_timezone_string();
```

Comment:
What happens if `wp_timezone_string` is defined, but the `timezone_string` option is not set for some reason? In that case we'll still need to fallback to the logic below, like we did before.

Reply:
It'll return either the timezone string or the GMT offset. `wp_timezone` and `wp_timezone_string` are meant to be a single API so we don't need to juggle which option is set. See https://make.wordpress.org/core/2019/09/23/date-time-improvements-wp-5-3/

Reply:
To add, check out the failing sync unit tests for how the return is different than previous. My major question is the output is slightly different looking at the failed unit tests—is this something that we can start using to simplify things or do we need to stick to the specific format that Sync massaged the data into before?

Reply:
Ah, I see, thanks for confirming. 

Well, I'd suggest refraining from modifying it for consistency reasons.

Reply:
Updated to revert out the changes to Sync and add a note on why it isn't using the modern method.

Reply:
Thank you!